From 4f33bc566c4cd83249fe71ef2e81b2eba439d1b7 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Mon, 24 Feb 2025 16:20:32 +0100
Subject: [PATCH 1605/1695] product: tfm-stm32mp2: update scmi agents file name

Rename stm32mp25-agents.h file into stm32mp2-agents.h to introduce
another stm32mp2xx platform.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git afd61a7697df9afc793a5b39cc01727e6bd75599 product: tfm-stm32mp2: update scmi agents file name]
Change-Id: I648a94bbedeebfda8dfcddc1faa86f7467217aaf
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/SCP-firmware/+/439638
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 .../SCP-firmware/product/tfm-stm32mp2/include/scmi_agents.h     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/include/scmi_agents.h b/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/include/scmi_agents.h
index be74d9b91..381d96a7d 100644
--- a/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/include/scmi_agents.h
+++ b/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/include/scmi_agents.h
@@ -7,7 +7,7 @@
 
 #ifndef SCMI_AGENTS_H
 #define SCMI_AGENTS_H
-#include <dt-bindings/scmi/stm32mp25-agents.h>
 
+#include <dt-bindings/scmi/stm32mp2-agents.h>
 
 #endif /* SCMI_AGENTS_H */
-- 
2.47.2

